manual tests/4 standalone binaries/build_osx_package.sh

```diff
@@ -8,13 +8,12 @@ rm -rf buildtmp
 mkdir -p mnttmp
 rm -f working.dmg
 gunzip < template.dmg.gz > working.dmg
-hdiutil attach working.dmg -noautoopen -quiet -mountpoint mnttmp
 # NOTE: output of hdiutil changes every now and then.
 # Verify that this is still working.
-DEV=`hdiutil info|tail -1|awk '{print $1}'`
+hdiutil attach working.dmg -noautoopen -quiet -mountpoint mnttmp
```

Comment:
This got moved after the comment, but hasn't actually changed and might not be related to the comment.